.github/workflows/python_ci.yml

```diff
@@ -28,12 +28,12 @@ jobs:
       - name: Lint with black
         run: |
           black --check --diff marching-squares/marching_squares
-          black --check --diff cloud-mask/cloud_mask
+          black --check --diff cloud-mask
       - name: Type check with mypy
         run: |
           mypy --strict --implicit-reexport marching-squares/marching_squares
-          mypy --strict --implicit-reexport cloud-mask/cloud_mask
+          mypy --strict --implicit-reexport cloud-mask
```

Comment:
For completeness, this can also be fixed by removing the `cloud-mask/__init__.py` file which is what marks `cloud-mask` as a package in the first place. This may lead to other issues, though. I've never tried coding in python without packages for organisation.